src/main/webapp/app/assessment/assessment-filters/assessment-filters.component.ts

```diff
@@ -13,12 +13,14 @@ enum FilterProp {
     selector: 'jhi-assessment-filters',
     templateUrl: './assessment-filters.component.html',
 })
-export class AssessmentFiltersComponent {
+export class AssessmentFiltersComponent implements OnChanges {
     FilterProp = FilterProp;
 
     filterProp: FilterProp = FilterProp.ALL;
 
-    @Input() submissions: Submission[] = [];
+    @Input()
```

Comment:
Why did you put `@Input()` on a separate line here? You did not do the same with the `@Output()` below.
Please keep this consistent within a single file.